packages/api-clients/open-api/purposeTemplateApi.yml

```
@@ -0,0 +1,1064 @@
+openapi: 3.0.3
+info:
+  title: Purpose Template Process Micro Service
+  description: Offers operations to retrieve purpose template data
+  version: "{{version}}"
+  contact:
+    name: API Support
+    url: "http://www.example.com/support"
+    email: support@example.com
+  termsOfService: "http://swagger.io/terms/"
+  x-api-id: an x-api-id
+  x-summary: an x-summary
+servers:
+  - url: "/"
+    description: This service is the purpose template process
+security:
+  - bearerAuth: []
+tags:
+  - name: purposeTemplate
+    description: Implements purpose template process
+    externalDocs:
+      description: Find out more
+      url: "http://swagger.io"
+  - name: health
+    description: Verify service status
+    externalDocs:
+      description: Find out more
+      url: "http://swagger.io"
+paths:
+  /status:
+    get:
+      security: []
+      tags:
+        - health
+      summary: Health status endpoint
+      description: Return ok
+      operationId: getStatus
+      responses:
+        "200":
+          description: successful operation
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+  /purposeTemplates:
+    parameters:
+      - $ref: "#/components/parameters/CorrelationIdHeader"
+    get:
+      tags:
+        - purposeTemplate
+      operationId: getPurposeTemplates
+      description: Retrieve Purpose Templates with pagination and filters
+      parameters:
+        - in: query
+          name: purposeTitle
+          description: filter by purpose title
+          schema:
+            type: string
+        - in: query
+          name: creatorIds
+          description: comma separated sequence of creator IDs
+          schema:
+            type: array
+            items:
+              type: string
+              format: uuid
+            default: []
+          explode: false
+        - in: query
+          name: states
+          description: filter by template state
+          schema:
+            type: array
+            items:
+              $ref: "#/components/schemas/PurposeTemplateState"
+            default: []
+          explode: false
+        - in: query
+          name: offset
+          required: true
+          schema:
+            type: integer
+            format: int32
+            minimum: 0
+        - in: query
+          name: limit
+          required: true
+          schema:
+            type: integer
+            format: int32
+            minimum: 1
+            maximum: 50
+      responses:
+        "200":
+          description: Purpose Templates requested
```

Minor: request is successful when you successfully **retrieve** data, not when you request
(same everywhere you use "requested", double-check)
```suggestion
          description: Purpose Templates retrieved
```